### PR TITLE
Refacto join channels with autocomplete

### PR DESCRIPTION
**Pull request by RkAiman**

### CODE REVIEW

File: twake/frontend/src/app/scenes/Client/ChannelsBar/Modals/WorkspaceChannelList.tsx [hunk 1]
```diff
@@ -24,36 +37,29 @@ export default () => {
...
-            <Typography.Link
-              style={{ color: 'var(--grey-dark)' }}
-              onClick={() => setLoadedChannels(loadedChannels + 3)}
-            >
-              {Languages.t('scenes.client.channelbar.channelmemberslist.loader')}
-            </Typography.Link>
-          </Row>
-        )}
+        {/* {!autoChannels.filter(({ name }) => name.toUpperCase().indexOf(search.toUpperCase()) > -1)
+          .length && setLimit(autoChannels.length + 2)} */}
```
Comment (RomaricMourgues) [inline twake/frontend/src/app/scenes/Client/ChannelsBar/Modals/WorkspaceChannelList.tsx:62]: Try something like this to not call the function setLimit on each render:
`setLimit(autoChannels.length + 2)` => `limit < autoChannels.length + 2 && setLimit(autoChannels.length + 2)`

Comment (RkAiman) [reply twake/frontend/src/app/scenes/Client/ChannelsBar/Modals/WorkspaceChannelList.tsx]: Done :+1:

File: twake/frontend/src/app/scenes/Client/ChannelsBar/Modals/WorkspaceChannelList.tsx [hunk 2]
```diff
@@ -24,36 +37,31 @@ export default () => {
...
-          <Row align="middle" justify="center" className="y-margin">
-            <Typography.Link
-              style={{ color: 'var(--grey-dark)' }}
-              onClick={() => setLoadedChannels(loadedChannels + 3)}
-            >
-              {Languages.t('scenes.client.channelbar.channelmemberslist.loader')}
-            </Typography.Link>
-          </Row>
-        )}
+        {!autoChannels.filter(({ name }) => name.toUpperCase().indexOf(search.toUpperCase()) > -1)
```
Comment (stephanevieira75) [inline twake/frontend/src/app/scenes/Client/ChannelsBar/Modals/WorkspaceChannelList.tsx:61]: Are you trying to capitalize first letter ? 

If yes, you can use `capitalize` from lodash.

https://lodash.com/docs/#capitalize

```js
import { capitalize } from 'lodash';
....

 {!autoChannels.filter(({ name }) => capitalize(name)
```

Comment (RomaricMourgues) [reply twake/frontend/src/app/scenes/Client/ChannelsBar/Modals/WorkspaceChannelList.tsx:61]: No, the goal here is to search for string into a string, regardless of case. So it is working.
@RkAiman can you implement something to also ignore accents in your next PR for members search ? (I send you a DM too)

**Pull request merged**